src/AzerothWarsCSharp.MacroTools/Spells/StormEarthandFire.cs

```diff
@@ -0,0 +1,53 @@
+using AzerothWarsCSharp.MacroTools.SpellSystem;
+using static War3Api.Common;
+using AzerothWarsCSharp.MacroTools.Libraries;
+
+namespace AzerothWarsCSharp.MacroTools.Spells
+{
+  public sealed class StormEarthandFire : Spell
+  {
+    public int UNIT_TYPE_1 { get; init; } = FourCC("npn4");
+    public int UNIT_TYPE_2 { get; init; } = FourCC("npn5");
+    public int UNIT_TYPE_3 { get; init; } = FourCC("npn6");
+    /// <summary>
+    /// How long the summoned units last.
+    /// </summary>
+    public float Duration { get; init; } = 60.0F;
+    public string EFFECT_TARGET { get; init; } = "Abilities\\Spells\\Items\\AIil\\AIilTarget.mdl";
+    public float EFFECT_SCALE_TARGET { get; init; } = 1.0F;
+
+    public float HEALTH_BONUS_BASE { get; init; } = -0.15F;   //These refer to the % extra amount of that stat a Simulacrum gets
```

Comment:
The comment about Simulacrums don't really make sense here. Either remove the comment or make it make sense.